wazo_calld/plugins/switchboards/confd.py

```diff
@@ -16,12 +17,24 @@ def __init__(self, tenant_uuid, uuid, confd_client):
 
     def exists(self):
         try:
-            self._confd.switchboards.get(self.uuid, tenant_uuid=self.tenant_uuid)
+            self._get_queue()
+        except NoSuchSwitchboard:
+            return False
+        else:
+            return True
+
+    def hold_moh(self):
+        return self._get_queue()['waiting_room_music_on_hold']
+
+    def queue_moh(self):
+        return self._get_queue()['queue_music_on_hold']
+
+    def _get_queue(self):
```

Comment:
I think that you refer to asterisk-queue, but to avoid confusion with wazo-queue
```suggestion
    def _get(self):
```
or
```suggestion
    def _get_switchboard(self):
```

Reply:
fixed